lib/terminology_table.erb

```diff
@@ -67,4 +73,4 @@
     </tr>
     <% end %>
   </tbody>
-</table>
+</table>
```

Comment:
add final newline back to file

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/blob/related-person-updates/lib/terminology_table.erb#L72)

Reply:
Just realized didn't really add a new line, for some reason I'm adding a new line but its not showing up in the actual commit.

Reply:
in the github UI it will display a red 'no trailing newline' icon at the end of files that don't have newlines when viewing diffs. doesn't seem that the icon is present any longer, so this looks good to me